setup.py

```diff
@@ -198,6 +199,9 @@ def run(self):
             osutil.stop_agent_service()
             osutil.start_agent_service()
 
+requires = []
+if float(sys.version[:3]) >= 3.5:
```

Comment:
Perhaps we can substitute the version (or even the check and version) for a constant like `PLATFORM_REMOVED` or `PLATFORM_AVAILABLE`

Reply:
Sorry I don't get this??

Reply:
Rather than having a specific version number with a comment, it might be more readable if we use a constant and/or helper method which indicates what the version check is for, something like:
```
if float() >= PLATFORM_DEPRECATED
```
or
```
if PLATFORM_SUPPORTED_VERSION
```
Just a thought for maintainability.